.evergreen/run-tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       AUTH                    Set to enable authentication. Defaults to "noauth"
+#       SSL                     Set to enable SSL. Defaults to "nossl"
+#       MONGODB_URI             Set the suggested connection MONGODB_URI (including credentials and topology info)
+#       MARCH                   Machine Architecture. Defaults to lowercase uname -m
+
+AUTH=${AUTH:-noauth}
+SSL=${SSL:-nossl}
+MONGODB_URI=${MONGODB_URI:-}
+DRIVERS_TOOLS=${DRIVERS_TOOLS:-}
+MONGODB_VERSION=${MONGODB_VERSION:-}
+
+# install MongoDB
+# Functions to fetch MongoDB binaries
+. ${DRIVERS_TOOLS}/.evergreen/download-mongodb.sh
+
+get_distro
+if [ -z "$MONGODB_DOWNLOAD_URL" ]; then
+    get_mongodb_download_url_for "$DISTRO" "$MONGODB_VERSION"
+fi
+# Even though we have the MONGODB_DOWNLOAD_URL, we still call this to get the proper EXTRACT variable
+get_mongodb_download_url_for "$DISTRO"
+download_and_extract "$MONGODB_DOWNLOAD_URL" "$EXTRACT"
+
+# run tests
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
```

Comment:
I feel that things like this could be made into bash functions to make it clearer what is happening.

Reply:
this is a straight rip of the existing mongo-orchestration scripts, soon to be replaced by work Sam Kleinman is doing with the `curator` method (read: this is going away soon).